docs/content/guides/developer/app-examples/escrow-frontend.mdx

```diff
@@ -0,0 +1,823 @@
+---
+title: Escrow Frontend
+sidebar_label: Frontend
+toc_max_heading_level: 2
+---
+
+:::note Multi-Page Guide
+
+This is the third in a [three-part guide](/guides/developer/app-examples/escrow) on how to build a trustless atomic swap on Sui.
+
+:::
+
+In this final part of the app example, you will build a frontend (UI) that allows end-users to discover trades and interact with listed escrows.
+
+## Prerequisites
+
+:::info
+
+You can view the [complete source code for this app example](https://github.com/MystenLabs/sui/tree/main/examples/trading) in the Sui repository.
+
+:::
+
+Before getting started, make sure you:
+
+- Understand [the mechanism behind the Escrow smart contract backend](/guides/developer/app-examples/escrow).
+- Check out [indexing service guide](/guides/developer/app-examples/escrow-indexer-api) to learn how we index on-chain data and API endpoints exposed to serve data query requests.
+- Install [`pnpm` through this guide](https://pnpm.io/installation) as we will use it as our package manager.
+- Check out [Sui Typescript SDK](https://sdk.mystenlabs.com/typescript) for basic usage on how to interact with Sui with Typescript.
+- Check out [Sui dApp Kit](https://sdk.mystenlabs.com/dapp-kit) to learn basic building blocks for developing a dApp in the Sui ecosystem with React.js.
+- Check out [React Router](https://reactrouter.com/en/main) as we use it to navigate between different routes in our UI website.
+- `dApp Kit` provides a set of hooks for making query and mutation calls to Sui blockchain. These hooks are thin wrappers around query and mutation hooks from `@tanstack/react-query`. Please check out [@tanstack/react-query](https://tanstack.com/query/latest/docs/framework/react/overview) to learn the basic usage for managing, caching, mutating server state.
+- This project is bootstrapped through `pnpm create @mysten/dapp`. Please check out [@mysten/create-dapp](https://sdk.mystenlabs.com/dapp-kit/create-dapp) for how to scaffold a React.js Sui dApp project quickly.
+
+## Overview
+
+Let's start with the UI wireframe.
```

Comment:
Is there an actual wireframe that will be added here?

Reply:
Same. It's referred to in a couple of places, too.

Reply:
We actually don't. My initial intention to describe the "imagination" UI wireframe so readers can have an overall sense of what the actual UI looks like. Maybe @ronny-mysten can help with my wording?